contrib/meson/meson.build

```diff
@@ -11,7 +11,7 @@ project('lz4', ['c'],
   license: ['BSD', 'GPLv2'],
   default_options : ['c_std=c99',
     'buildtype=release'],
-  version: '1.8.3',
+  version: '1.9.1',
```

Comment:
Do you believe there could be a way to extract the version number from `lz4.h` directly ?
That's the way the `Makefile` works currently.

Reply:
It already extract version number from `lz4.h`. This is just a fallback
project version in case extracting version number failed.

Reply:
Is there a way to add a comment which would explain that ?

Reply:
Done.

Reply:
I would suggest failing if the version cannot be extracted instead of having a fallback value.

Reply:
Currently, meson build script compares extracted version and
default fallback version. If extracted is smaller than the default, meson will
use default version. Otherwise it'll use extracted version.
Do you prefer to fail in case extracted version is smaller than fallback version
as @devnoname120 suggest, @Cyan4973 ?

Reply:
I believe @devnoname120 suggests to _only rely on extracted version_, and have no manually set fallback version.

While I don't think strongly about it, I would rather lean towards @devnoname120 's suggestion.

This way, if version extraction fails, we immediately have a problem, hence an alert,
which we can act upon.

Reply:
Thanks for the feedback, I will change this PR soon.

Reply:
Done.